app/app.js

```diff
@@ -0,0 +1,18 @@
+const express = require('express')
+const {
+    handleCustomErrors,
+    handlePsqlErrors,
+    handleServerErrors,
+} = require('./errors/index.js');
+const { getAllTopics } = require('../app/controllers/topics.controller.js')
+const app = express()
+
+app.get('/api/topics', getAllTopics)
+
+
+app.use(handleCustomErrors);
+app.use(handlePsqlErrors);
```

Comment:
we don't need any psql errors yet, try not to add functionality not needed for this ticket